protecode/util.py

```diff
@@ -343,6 +347,14 @@ def upload_result_to_compliance_issue(
     issue = dso.model.ComplianceIssue(
         artefact=artefact_ref,
         meta=meta,
-        data={'greatestCvss3Score': greatest_cvss3_score},
+        data={
+            'greatestCvss3Score': greatest_cvss3_score,
+            'protecodeProductUrl': ci.util.urljoin(
```

Comment:
I think it should be possible to read this url from either the protecode-results, or using the protecode-client, should it not?

Reply:
had to extend the model, product url is now taken from protecode api response :)